platform/security/common/src/main/java/org/codice/ddf/security/common/jaxrs/RestSecurity.java

```diff
@@ -46,13 +42,14 @@
  */
 public final class RestSecurity {
 
-    // SAML_COOKIE_NAME is not available in SecurityConstants 2.4.0
-    public static final String SECURITY_COOKIE_NAME = "org.codice.websso.saml.token";
```

Comment:
It won't be incompatible, you'll still be able to federate and pass the machine's identity. You just won't be able to federate the user's identity between an old and new DDF. We should probably just document that this change happened @garrettfreibott and that federating a user's identity between a 2.7.x and 2.8.x via REST won't work.